packages/zoe/test/swingsetTests/zoe/test-zoe.js

```diff
@@ -285,6 +286,26 @@ test.serial('zoe - sellTickets - valid inputs', async t => {
   t.deepEqual(dump.log, expectedSellTicketsOkLog);
 });
 
+const expectedOTCDeskOkLog = [
+  '=> alice, bob, carol and dave are set up',
+  'Inventory added',
+  '@@ schedule task for:1, currently: 0 @@',
+  'Trade successful',
+  '{"brand":{},"value":3}',
+  '{"brand":{},"value":0}',
+  'Inventory removed',
+  '{"brand":{},"value":2}',
+];
+test.serial('zoe - otcDesk - valid inputs', async t => {
+  t.plan(1);
```

Comment:
I thought we didn't need `t.plan()` with AVA. Why is this needed?

Reply:
I was cargo-culting (actually, more like copy-and-pasting). It looks like it got partially removed from the other tests in this file but not all. It is not necessary.